test/e2e/scheduling/preemption.go

```diff
@@ -669,6 +669,84 @@ var _ = SIGDescribe("SchedulerPreemption [Serial]", func() {
 			}
 		})
 	})
+
+	ginkgo.Context("PriorityClass endpoints", func() {
+		var cs clientset.Interface
+		f := framework.NewDefaultFramework("sched-preemption-path")
+		var pcs []*schedulingv1.PriorityClass
+
+		ginkgo.BeforeEach(func() {
+			cs = f.ClientSet
+			// Create 2 PriorityClass: p1, p2.
+			for i := 1; i <= 2; i++ {
+				name, val := fmt.Sprintf("p%d", i), int32(i)
+				pc, err := cs.SchedulingV1().PriorityClasses().Create(context.TODO(), &schedulingv1.PriorityClass{ObjectMeta: metav1.ObjectMeta{Name: name}, Value: val}, metav1.CreateOptions{})
+				if err != nil {
+					framework.Logf("Failed to create priority '%v/%v': %v", name, val, err)
+					framework.Logf("Reason: %v. Msg: %v", apierrors.ReasonForError(err), err)
+				}
+				framework.ExpectEqual(err == nil || apierrors.IsAlreadyExists(err), true)
+				pcs = append(pcs, pc)
+			}
+		})
+
+		ginkgo.AfterEach(func() {
+			// Print out additional info if tests failed.
+			if ginkgo.CurrentGinkgoTestDescription().Failed {
+				// list existing priorities
+				priorityList, err := cs.SchedulingV1().PriorityClasses().List(context.TODO(), metav1.ListOptions{})
+				if err != nil {
+					framework.Logf("Unable to list priorities: %v", err)
```

Comment:
"priorities" is good,  but "PriorityClasses" might be better for consistency.

Reply:
done.